Marlin/Marlin_main.cpp

```diff
@@ -7258,6 +7258,14 @@ inline void gcode_M42() {
 
 #endif // AUTO_BED_LEVELING_UBL && UBL_G26_MESH_VALIDATION
 
+/**M73: Set LCD Progress Bar Percentage */ 
+#if ENABLED(LCD_PROGRESS_BAR) 
+  inline void gcode_M73() { 
+  #include "ultralcd_impl_HD44780.h" 
```

Comment:
These includes are dangerous here, and should only ever be included at the top of `ultralcd.cpp`. I like the concept, but this is not the correct implementation. Let me see if I can draw something up that shows the correct methodology.

Reply:
Adding the labels "Feature Request" and "Needs Work" is something I think should happen